index.html

```diff
@@ -7,19 +7,20 @@
 
     <link rel="stylesheet" type="text/css" href="index.css">
     <link href="https://fonts.googleapis.com/css?family=Lato|Raleway&display=swap" rel="stylesheet">
+    <link href='https://actionnetwork.org/css/style-embed-whitelabel-v3.css' rel='stylesheet' type='text/css' />
   </head>
   <body>
     <!-- This webpage is setup to have an anchored image on the left side and scrolling content on the right  -->
     <div class="panel" id="static">
       <img id="anchor_img" src="indigenous_protectors.jpg" />
     </div>
 
-    <div class="panel" id="dynamic">
+    <div class="panel" id="scrolling">
       <div id="main_content">
         <div class="section">
-          <h1 id="call-to-action">CALL TO ACTION/DIRECT APPEAL HERE</h1>
-
-          <h2>Declare Climate Emergency</h2>
+          <h1 id="call-to-action">Declare Climate Emergency in Your State or Local Government</h1>
+          <hr />
```

Comment:
Some styling on this would be good I think. The line is too strong right now, I think it needs to be a gray or turn down the opacity as opposed to the default black.

Reply:
Referring to `<hr/>` that is.